src/server/apis/issues.ts

```diff
@@ -109,6 +123,19 @@ query AssignedIssues($query: String!){
           }
         }
         ...popularityFields
+        ...commentFields
+      }
+    }
+  }
+}
+
+fragment commentFields on Issue {
+  updatedAt
+  commentFields: comments(last: 1) {
+    nodes {
+      updatedAt
```

Comment:
Changed these to `createdAt`. I probably don't want to see the notification when you update the issue/latest comment.